daemon/volumes.go

```diff
@@ -27,9 +28,12 @@ type mounts []container.Mount
 
 // volumeToAPIType converts a volume.Volume to the type used by the Engine API
 func volumeToAPIType(v volume.Volume) *types.Volume {
+	createdAt, _ := v.CreatedAt()
+	createdAtStringFormat := createdAt.Format(time.RFC3339)
 	tv := &types.Volume{
-		Name:   v.Name(),
-		Driver: v.DriverName(),
+		Name:      v.Name(),
+		Driver:    v.DriverName(),
+		CreatedAt: createdAtStringFormat,
```

Comment:
Same for me; no blocker, but would be nice to update.

(Also note that In "Go" convention, naming of variables usually depends on the "distance" between the variable being defined and used. "short distance"; use a short variable name, "long distance"; use a more descriptive name. In this case it's a short distance, so a short variable name would be more suitable instead of a quite verbose one)